pluto_build/sql/qaqc_housing_units.sql

```diff
@@ -0,0 +1,45 @@
+-- Create qaqc table All PLUTO records where there is a match in Housing Database 
```

Comment:
Missing "of"
I think this need to be " Create qaqc table of all PLUTO records where there is a match in Housing Database"